os/arch/xtensa/src/esp32/esp32_i2c.c

```diff
@@ -646,21 +677,21 @@ static void i2c_esp32_get_i2c_data(struct esp32_i2c_priv_s *priv)
 0:  no slave address
 1: 7bit slave address
 2: 10bit slave address*/
-static void i2c_esp32_send_write_cmd(struct esp32_i2c_priv_s *priv, int addr_bytes, int cmd_start_index)
+static int i2c_esp32_send_write_cmd(struct esp32_i2c_priv_s *priv, int addr_bytes, int cmd_start_index)
 {
 	struct i2c_msg_s *msg = priv->transfer_msg;
-	uint32_t i = 0;
+	uint32_t i, to_send = 0;
```

Comment:
please declare new variable in new line.

like: 
           uint32_t i;
           uint32_t to_send = 0;

Reply:
Fixed, thanks.